crate_universe/Cargo.toml

```diff
@@ -11,7 +11,12 @@ version = "0.9.0"
 authors = [
     "Andre Brisco - andre.brisco@protonmail.com",
 ]
+categories = ["development-tools"]
+description = "A collection of tools which use Cargo to generate build targets for Bazel"
```

Comment:
This wording is copied from the readme.

https://github.com/bazelbuild/rules_rust/blob/6b50aac86529cecc0053fded8ecf3914af33de7c/crate_universe/README.md#L3